wireless_modules/back_module/main.py

```diff
@@ -13,34 +15,52 @@
 MODULE_NUM = "3"
 
 # RZERO (for calibration of MQ135) found when the sensor was first 'activated'
-RZERO = 8.62
+RZERO = 6.1
```

Comment:
Also, maybe I wasn't as clear initially, but should we document in notion perhaps, the `RZERO` value for each sensor we have calibrated. In case we end up using the sensor I used initially, we don't want to do the 24-hour heat up and measure this again. So it might be good to map what the `RZERO` value is for each sensor we have calibrated.

Reply:
That's definitely a good idea!

For the time being I've reverted changing the RZERO in this PR, since I've realised my testing wasn't valid since I did it in my room rather than somewhere ventilated.